packages/react-component-library/src/components/TopLevelNavigation/NotificationPanel/Notification.tsx

```diff
@@ -45,55 +53,46 @@ export const Notification: React.FC<NotificationProps> = ({
   const contentId = getId('content')
 
   return (
-    <li data-testid="notification" {...rest}>
-      <div className="rn-notifications-item-wrapper">
+    <StyledNotification data-testid="notification" {...rest}>
+      <StyledWrapper>
         {React.cloneElement(link as ReactElement, {
           ...link.props,
           children: (
-            <div
+            <StyledItem
               aria-describedby={contentId}
-              className="rn-notifications-item"
               data-testid="notification-row"
               role="row"
             >
-              <div className="rn-notifications-item__avatar" role="gridcell">
+              <StyledAvatar role="gridcell">
                 <Avatar
                   initials={getInitials(name)}
                   variant={AVATAR_VARIANT.DARK}
                 />
                 {!isRead && (
-                  <span
-                    className="rn-notification-panel__not-read rn-notification-panel__not-read--notification-item"
+                  <StyledNotRead
```

Comment:
I wonder if a nice way to do this could be to have a `StyledNotReadNotificationItem` (or find a better name 😂 ):

```
export const StyledNotReadNotificationItem = styled(StyledNotRead)`...`
```